src/test/resources/demo-counterexamples/cyclic-ref.vpr

```diff
@@ -0,0 +1,10 @@
+field next: Ref
```

Comment:
Wouldn't it make sense to integrate the same unit test annotations as used in Gobra to Silicon and check them for these test cases?

Reply:
Yes, but this would add a dependency to this Pull request.
And it would have to be implemented.